docs/exercise-201/exercise1.md

```diff
@@ -0,0 +1,106 @@
+### Configure Users permissions to access Argo CD
+
+#### Overview
+
+Argo CD provides a integration with many identity providers to authenticate and authorize users to interact with the UI and the API. In this exercise we are going to create a local user with no permission and gradually grant access to certain features.
+
+#### Exercises
+
+1. Local users are defined in the main Argo CD configmap (`argocd-cm`). Run the following command to create a new user (`workshop`) in Argo CD:
+
+```
+kubectl apply -f manifests/ArgoCD201-ArgoCDRBAC/argocd-cm.yaml
+```
+
+2. We need to login with Argo CD CLI so we can manage our newly created user. First we need to retrieve the admin password with:
+
+```
+kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d
```

Comment:
change to 
`kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d; echo`